sumologic_collectd_metrics/metrics_writer.py

```diff
@@ -51,7 +51,12 @@ def write_callback(self, raw_data, _=None):
         Write callback
         """
 
-        metrics = convert_to_metrics(raw_data, self.met_config.types)
+        try:
+            data_set = self.collectd.get_dataset(raw_data.type)
```

Comment:
Do we need to handle the `None` case as well as mentioned in `get_datset` return values here https://collectd.org/documentation/manpages/collectd-python.5.shtml  ?

Reply:
`min` and `max` has not been taken into account so far, and in case dataset gets invalid metric type it returns `TypeError`

Reply:
I open new issue to track this: #36